scripts/helm/install

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+set -e
+projectRoot="$(a="/$0"; a=${a%/*}; a=${a:-.}; a=${a#/}/; cd "$a/../.." || return; pwd)"
+
+(
+  cd "${projectRoot}"
+
+  echo "Deploying claim message queue"
+  helm repo add activemq-artemis https://mailrucloudsolutions.github.io/activemq-artemis-helm/
+  helm upgrade \
+    --atomic \
+    --install \
+    --namespace mine-support-claim-service \
+    --set artemisUser=artemis \
+    --set artemisPassword=artemis \
+    --set replicas=1 \
+    --set persistence.enabled=false \
+    mine-support-claim-artemis \
```

Comment:
the values overriden by `set` could be put in a yaml file rather than be set on the command line for the artemis, postgres and claim service

Reply:
Done.